proto/ibc/core/channel/v1/channel.proto

```diff
@@ -132,6 +132,18 @@ message PacketState {
   bytes data = 4;
 }
 
+// PacketId is an identifer for a unique Packet
+message PacketId {
+  option (gogoproto.goproto_getters) = false;
+
+  // channel port identifier.
+  string port_id = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
+  // channel unique identifier.
+  string channel_id = 2 [(gogoproto.moretags) = "yaml:\"channel_id\""];
```

Comment:
I think for consistency we should always specify this is **source** port and channel. cc: @colin-axner @ethanfrey

Reply:
That would mean a packet is only uniquely identifiable on the source chain, which is the case for 29-fee, but then this seems less compelling to bring into 04-channel since then this is really packet id as identified on the source chain

Why can't the destination chain refer to a packet using the destination port and channel?

Reply:
I mean both sides get access to source and dest port and channelID. So I think both sides using the same semantics will avoid confusion.

Even a destination chain can refer to packet using its source (counterparty) port/channel. It's mainly about keeping very consistent semantics

Reply:
Oh I see what you're saying, it's possible for collisions on dest chains. So we should make the semantics that the portID/channelID of the PacketID always refers to the port/channel on the chain in question

So source chains refer to packets by source port/channel
dest chains refer to packets by dest port/channel.

Yea let's make this the consistent rule. Good catch @colin-axner

Reply:
Updated the comment